libraries/python/assistant-drive/assistant_drive/__init__.py

```diff
@@ -1,6 +1,7 @@
-from .drive import Drive, DriveConfig
+from .drive import Drive, DriveConfig, IfExistsBehavior
```

Comment:
Not really loving the name of the enum when I would import this in my assistant code amongst all of the other imports.  Maybe something more like IfDriveFileExistsBehavior to be more clear?